contracts/Datastore.sol

```diff
@@ -267,24 +315,39 @@ contract Datastore is AragonApp {
     }
 
     /**
-     * Sets IPFS as the storage provider for the datastore.
+     * @notice Change the encryption provider
+     * @param _encryptionProvider Encryption provider
+     */
+    function setEncryptionProvider(EncryptionProvider _encryptionProvider) public {
```

Comment:
If the provider is changed, what happens with all the already existing files? Are they going to be encrypted? Is it possible to have unencrypted and encrypted files in the datastore simultaneously?

Reply:
Yes what we have in mind (at least in the first couple releases) is to lock the choice of encryption and of storage provider when they are initially selected. also it is possible to have encrypted and unencrypted files in the datastore

Reply:
But this isn't enforce yet, right? Where is this function being used?
Besides, the same comment could apply to `setEncryptionKey`. What happens if the file was already encrypted?

Reply:
Alright `setEncryptionProvider` and `setStorageProvider` are now removed: #91

Reply:
@macor161 could you cherry-pick that one here too? thanks!

Reply:
Those functions are now also removed in this pull request.